core/modules/replica/SqlApp.h

```diff
@@ -30,8 +30,8 @@ namespace qserv {
 namespace replica {
 
 /**
- * Class SqlApp implements a tool which executes the same query
- * against worker databases of select workers. Result sets
+ * Class SqlApp implements a tool which executes the same SQL
+ * statement against worker databases of select workers. Result sets
  * will be reported upon a completion of the application.
```

Comment:
Yes, you asked 😹 And most use cases for the initial design have been already addressed by the targeted `Sql<DoSomethingSpecific>Request` and `Sql<DoSomethingSpecific>Job` supported by the `Configuration` options to avoid passing user names and passwords over the wire of the controller-workers protocol. I still have at least one use case left which might be eliminated soon.